### DISCUSSION TITLE
How can I access the administration page?

### DISCUSSION THREAD

**Discussion by mariushosting** [BoidCMS/BoidCMS#7] [Q&A] [answered]
I have installed this on my own server.
How can I access my administrator page?

https:/mydomain.com/where is the admin page location?

**Comment by sysadev** [comment, marked as answer]
Hi @mariushosting, check the [installation steps](https://boidcms.github.io/#/install?id=default-login-credentials).      

> The default login page for the administrator is located at `admin`. To access the login page, simply append `/admin` to the end of your website's URL. For example, if your site's URL is `http://example.com`, then the login page would be located at `http://example.com/admin`. If your website is located in a subdirectory, then you will need to include the subdirectory name before `/admin`.
> 
> It's important to note that the default username and password are both set to **admin** and **password**, respectively. However, to ensure your site's security, it's highly recommended that you change these login details to something more unique and complex.
